tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from OMPython import OMParser
```

Comment:
Yeah, i was primarily wondering about i.porting it in __init__.py, then not using it.